scripts/release-checksums.rb

```diff
@@ -25,8 +30,10 @@ def get_sha256_checksum(url)
 OS | Arch | Package | SHA256 Checksum
 -- | -- | -- | --
 Windows | x64 | [exe](#{windows_exe}) | <code>#{windows_exe_sha256}</code>
-macOS | x64 | [zip](#{macos_zip}) | <code>#{macos_zip_sha256}</code>
-macOS | x64 | [DMG](#{macos_dmg}) | <code>#{macos_dmg_sha256}</code>
+macOS | x64 | [zip](#{macos_x64_zip}) | <code>#{macos_x64_zip_sha256}</code>
+macOS | arm64 | [zip](#{macos_arm64_zip}) | <code>#{macos_arm64_zip_sha256}</code>
+macOS | x64 | [DMG](#{macos_x64_dmg}) | <code>#{macos_x64_dmg_sha256}</code>
+macOS | arm64 | [DMG](#{macos_arm64_dmg}) | <code>#{macos_arm64_dmg_sha256}</code>
 Linux | x64 |  [AppImage](#{linux_appimage}) | <code>#{linux_appimage_sha256}</code>
```

Comment:
It's my fault that I missed mentioning the counterpart in CI https://github.com/nervosnetwork/neuron/blob/develop/.github/workflows/check_checksums.yml#L22, since there're 2 more records, the truncation should cover 2 more lines of the release note.

Reply:
Sorry for adding more details late.

The code
```
          python3 -c "import sys, json; print(\"\n\".join(json.load(sys.stdin)['body'].splitlines()[-9:]))" |\
```
in https://github.com/nervosnetwork/neuron/blob/develop/.github/workflows/check_checksums.yml#L22 will truncate last 9 lines of a release page, which is
![image](https://user-images.githubusercontent.com/7271329/198267622-517ca226-0d32-4924-826f-3820cc49d27a.png)

The code will be executed after a release to check if the checksums listed in the release note are consistent with the attached packages.

Now we're going to have 2 more attached packages, we will add 2 more checksums in the release note, so the code should truncate last 11 lines instead of 9 lines.